api/server.spec.js

```diff
@@ -0,0 +1,73 @@
+const request = require('supertest');
+
+const server = require('./server');
+const db = require('../data/dbConfig');
+
+describe('server.js', () => {
+  beforeEach(async () => {
+    await db('boardgames').truncate();
+  });
+
+  describe('GET /', () => {
+    it('status code should be 200 OK', async () => {
+      const res = await request(server).get('/');
+      expect(res.status).toBe(200);
+    });
+    it('res.type should be JSON', async () => {
+      const res = await request(server).get('/');
+      expect(res.type).toBe('application/json');
+    });
+    it("res.body should equal { api: 'running...' }", async () => {
+      const res = await request(server).get('/');
+      expect(res.body).toEqual({ api: 'running...' });
+    });
+  });
```

Comment:
all valid tests, although I thought you'd be testing your endpoint that actually returned your boardgames